code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -156,6 +156,16 @@
 	can_suppress = TRUE
 	mag_type = /obj/item/ammo_box/magazine/internal/cylinder/rev762
 
+/obj/item/gun/projectile/revolver/overgrown
+	name = "overgrown revolver"
+	desc = "A bulky revolver that seems to be made out of a plant."
+	icon_state = "pea_shooter"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
```

Comment:
Gun on the items_hand_bloat.dmi instead of gun_hand.dmi?

![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/9dade173-1ac7-473c-9b0c-82a2e62df0ef)

Reply:
Ah, the thingy I didn't know existed ... oops, will change